src/components/CodeBlock/CodeBlock.tsx

```diff
@@ -0,0 +1,174 @@
+import { HTMLAttributes, useRef, useState } from "react";
+import SyntaxHighlighter, { createElement } from "react-syntax-highlighter";
```

Comment:
I am pretty sure this will include all implementations in the bundle. I think we should create a file where we import only what we need.

Reply:
We can reduce the package size by importing the light version.
But we have to import only the necessary languages
Should we discuss the languages we should allow?

Reply:
@serdec @fneves Would like you thoughts here

Reply:
I would support only what we are currently supporting at the moment and yeah would import only the light version.
For now I would simply support SQL + JSON. 
Later on I would say we can expose a global configuration hook that allows us to register additional ones.